pandas/_libs/src/librdata/win_iconv.c

```diff
@@ -0,0 +1,2228 @@
+/*
+ * No Copyright.
```

Comment:
Perhaps license CC0 is cleaner?

Reply:
How about this using [CC0 wiki format](https://wiki.creativecommons.org/wiki/CC0_FAQ#May_I_apply_CC0_to_computer_software.3F_If_so.2C_is_there_a_recommended_implementation.3F)? May need `copyright` somewhere to pass `cpplint`.

> win-iconv - iconv implementation using Win32 API to convert.
Written in 2009 by Yukihiro Nakadaira (ynkdir) <<https://github.com/ynkdir>> and 
> Contributors to win-iconv <<https://github.com/win-iconv/win-iconv>>
>
> To the extent possible under law, the author(s) have dedicated all copyright and related and neighboring rights to this software to the public domain worldwide. This software is distributed without any warranty.
You should have received a copy of the CC0 Public Domain Dedication along with this software. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.